blaze/compute/pandas.py

```diff
@@ -165,6 +165,21 @@ def compute_up(expr, data, **kwargs):
     return compute_up(expr._child.distinct().count(), data, **kwargs)
 
 
+@dispatch(Repeat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.repeat(data)
+
+
+@dispatch(Concat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.cat(data)
```

Comment:
`expr.lhs` is an expression or a scalar value, you need to implement these operations similar to how you implemented the numpy backend version. Something like this:

```python
@dispatch(Concat, pd.Series)
def compute_up(t, data, **kwargs):
    if isinstance(t.lhs, Expr):
        return data + t.rhs
    else:
        return t.lhs + data
```
better yet, you can probably just not implement this, since `Concat` is a subclass of `Arithmetic` and the `BinOp` implement will suffice